pkg/zng/record.go

```diff
@@ -183,16 +183,11 @@ func (r *Record) ZeekStrings(precision int, utf8 bool) ([]string, bool, error) {
 			if err != nil {
 				return nil, false, err
 			}
-			sec, ns := ts.Split()
-			if precision == 6 && isHighPrecision(ns) {
+			if precision == 6 && isHighPrecision(ts) {
```

Comment:
removing the use of Split() here makes sense... ZeekStrings shouldn't  be fiddling around with that